app/localization/translated/ru.json

```diff
@@ -55,6 +55,12 @@
   "FiltersPage.msgFavoriteFilters": "Фильтры - это поисковые запросы, которые были сохранены для повторного использования.",
   "FiltersPage.searchByName": "Поиск по Имени",
   "FiltersPage.title": "Фильтры",
+  "Footer.build": "Сборка",
+  "Footer.allrights": "Все права защищены",
+  "Footer.git": "Мы на GitHub",
+  "Footer.slack": "Пообщайтесь с нами на Slack",
+  "Footer.contact": "Свяжитесь с нами",
+  "Footer.documentation": "Документация",
```

Comment:
Please also add Belorussian localization. Translations you can find here https://github.com/reportportal/service-ui/blob/localization_by/src/main/resources/public/js/src/localizations/be-BY.js